packages/shared/ReactSymbols.js

```diff
@@ -32,6 +33,9 @@ export const REACT_OFFSCREEN_TYPE = Symbol.for('react.offscreen');
 export const REACT_LEGACY_HIDDEN_TYPE = Symbol.for('react.legacy_hidden');
 export const REACT_CACHE_TYPE = Symbol.for('react.cache');
 export const REACT_TRACING_MARKER_TYPE = Symbol.for('react.tracing_marker');
+export const REACT_SERVER_CONTEXT_DEFAULT_VALUE_NOT_LOADED = Symbol.for(
+  'react.server_context.defaultValue',
```

Comment:
This is pretty long (more bytes) and the other ones use all lower case. Maybe `react.default_value`?